### PR TITLE
#1: Functionality to Click on product items and view their details

### PR DESCRIPTION
**Pull request by MuindiStephen**

### CODE REVIEW

**Review by MuindiStephen — COMMENTED** [review]
DONE

File: gradle/libs.versions.toml
Comment (MuindiStephen) [inline gradle/libs.versions.toml:1]: Bump AGP version for compatibility. AGP 8.5.0

**Review by MuindiStephen — COMMENTED** [review]
OK

**Review by OkelloSam21 — APPROVED** [review]

**Review by OkelloSam21 — APPROVED** [review]

**Review by OkelloSam21 — APPROVED** [review]
Nice work 🎉🙌

**Pull request merged**